### PR TITLE
Swerve2022: Cleaning Up Swerve Code

### PR DESCRIPTION
**Pull request by hpark-24**

### CODE REVIEW

**Review by amull23 — APPROVED** [review]
looks great!

File: Swerve2022/src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -12,25 +14,38 @@
...
+    public static final int    kLRSteer6CANID         = 6;
+    public static final int    kRRDrive7CANID         = 7;
+    public static final int    kRRSteer8CANID         = 8;
 
-    public static final int kLFDrive1CANID = 1;
-    public static final int kLFSteer2CANID = 2;
-    public static final int kRFDrive3CANID = 3;
-    public static final int kRFSteer4CANID = 4;
+    public static final double kWheelDiaMeters        = Units.inchesToMeters(4.0); // 4in (39.37 in/meter)
+    public static final double kEncoderMetersPerCount = (kWheelDiaMeters * Math.PI) / (Falcon500.kEncoderCPR) / 6.75;
```
Comment (jmullins16) [inline Swerve2022/src/main/java/frc/robot/Constants.java:48]: Let's make another named constant for the gear ratio, so the next person knows what that 6.75 magic number is for. I think we did that in the Offseason2022 project.

**Comment by reevar24** [comment]
Since we're not using PWMSparkMax, I think it would be okay to delete the import statement for that on line 17 in SwerveModule.java

**Review by jmullins16 — CHANGES_REQUESTED** [review]
Looks like it would function correctly as is. Just a couple requests to make it easier for the next person to follow. Already identified the gear ratio item. For the steer/turn motors, we should make the named constants and object naming match. If you prefer turnMotor... then let's make the constants be kLFTurnMotorCANID for consistency. Just locks in the reader's mind that they go together.

Nice work!

File: Swerve2022/src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (hpark-24) [reply Swerve2022/src/main/java/frc/robot/Constants.java]: I'm not quite sure what I need to change for having the steer/turn motors constants and object naming matching.

Comment (jmullins16) [reply Swerve2022/src/main/java/frc/robot/Constants.java]: Just change the name of the kXXSteerNCANID literals to be kXXTurnNCANID (where XX is LF, RF, etc. and N is the CAN ID value). 

Then when SwerveModule is called, the parameter name will match the constant being passed.

Comment (jmullins16) [reply Swerve2022/src/main/java/frc/robot/Constants.java]: Looks good now.

**Review by jmullins16 — APPROVED** [review]

**Pull request merged**